csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1779,95 +1896,104 @@ CSAF documents are based on JSON, thus the security considerations of [RFC8259]
-* **CSAF post-processor**: A CSAF producer that transforms an existing CSAF document into a new CSAF document, for example, by removing or redacting security-sensitive elements.
+* **CSAF post-processor**: A CSAF producer that transforms an existing CSAF document into a new CSAF document, for example, by removing or redacting elements according to sharing policies.
 * **CSAF modifier**: A CSAF post-processor which takes a CSAF document as input and modifies the structure or values of properties. The output is a valid CSAF document.
 * **CSAF translator**: A CSAF post-processor which takes a CSAF document as input and translates values of properties into another language. The output is a valid CSAF document.
 * **CSAF consumer**: A program that reads and interprets a CSAF document.
 * **CSAF viewer**: A CSAF consumer that reads a CSAF document, displays a list of the results it contains, and allows an end user to view each result in the context of the artifact in which it occurs.
 * **CSAF management system**: A program that is able to manage CSAF documents and is able to display their details as required by CSAF viewer.
 * **CSAF asset matching system**: A program that connects to or is asset database and is able to manage CSAF documents as required by CSAF management system as well as matching them to assets of the asset database.
 
-## 5.2 Conformance Clause 1: CSAF document
+### 5.1.1 Conformance Clause 1: CSAF document
+
+A text file satisfies the "CSAF document" conformance profile if the text file:
 
-A text file satisfies the "CSAF document" conformance profile if:
+* conforms to the syntax and semantics defined in section 3.
 
-* It conforms to the syntax and semantics defined in section 3.
+### 5.1.2 Conformance Clause 2: CSAF producer
 
-## 5.3 Conformance Clause 2: CSAF producer
+A program satisfies the "CSAF producer" conformance profile if the program:
 
-A program satisfies the "CSAF producer" conformance profile if:
+* produces output in the CSAF format, according to the semantics defined in section 3.
+* satisfies those normative requirements in section 3 that are designated as applying to CSAF producers.
 
-* It produces output in the CSAF format, according to the semantics defined in section 3.
-* It satisfies those normative requirements in section 3 that are designated as applying to CSAF producers.
+### 5.1.3 Conformance Clause 3: Direct producer
 
-## 5.4 Conformance Clause 3: Direct producer
+An analysis tool satisfies the "Direct producer" conformance profile if the analysis tool:
 
-An analysis tool satisfies the "Direct producer" conformance profile if:
+* satisfies the "CSAF producer" conformance profile.
+* additionally satisfies those normative requirements in section 3 that are designated as applying to "direct producers" or to "analysis tools".
+* does not emit any objects, properties, or values which, according to section 3, are intended to be produced only by converters.
```

Comment:
Such normative requirements do not exist yet in section 3 (if I remember correctly). Do we want to introduce them?

Reply:
I guess the best would be to refer to section 5 as we have done in the other cases.